src/components/Project/Project.js

```diff
@@ -191,6 +241,71 @@ const Project = ({
     [handleSetProjectData]
   )
 
+  const handleAddProjectLabel = (label, labels) => {
+    const objectLabels = {}
+
+    labels.forEach(label => {
+      const splitedLabel = label.split(':')
+      objectLabels[splitedLabel[0]] = splitedLabel[1].replace(' ', '')
+    })
+
+    const newLabel = {
+      [label.split(':')[0]]: label.split(':')[1].replace(' ', '')
+    }
+
+    setVisibleChipsMaxLength(null)
+    addProjectLabel(newLabel, objectLabels)
+  }
+
+  const handleUpdateProjectLabels = labels => {
+    const objectLabels = {}
+
+    labels.forEach(label => {
+      const splitedLabel = label.split(':')
+
+      objectLabels[splitedLabel[0]] = splitedLabel[1].replace(' ', '')
+    })
+
+    const projectData = {
+      description:
+        editProject.description.value ??
+        projectStore.project.data.spec.description,
+      goals: editProject.goals.value ?? projectStore.project.data.spec.goals,
+      name: editProject.name.value ?? projectStore.project.data.metadata.name,
+      source: editProject.source.value ?? projectStore.project.data.spec.source
+    }
+    const data = {
+      ...projectStore.project.data,
+      metadata: {
+        ...projectStore.project.data.metadata,
+        labels: objectLabels,
+        name: projectData.name
+      },
+      spec: {
+        ...projectStore.project.data.spec,
+        description: projectData.description,
+        goals: projectData.goals,
+        source: projectData.source
+      }
+    }
+
+    setVisibleChipsMaxLength(1)
+
```

Comment:
don't add empty lines after function call if there is also function call next to it